admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -83,6 +83,7 @@
 @import "pages/improve";
 @import "pages/payment";
 @import "pages/translation_page";
+@import "pages/product_preferences";
```

Comment:
I think you have the product page own bundle no? I did the split some months ago so it's lighter for every other pages

But maybe not related to the product page v2

Reply:
Not related. This one is for `configure/shop/preferences/preferences` page